src/components/Contacts/ContactListTableMobile.jsx

```diff
@@ -0,0 +1,199 @@
+import { useTheme } from '@mui/material/styles';
+// Custom Hooks Imports
+import useNotification from '@hooks/useNotification';
+// Util Imports
+import { saveToClipboard } from '@utils';
+// Component Imports
+import ContactProfileIcon from './ContactProfileIcon';
+
+/**
+ * @typedef {object} Contact
+ * @property {string} webId - The Web ID of the contact
+ * @property {string} givenName - The given name of the contact
+ * @property {string} familyName - The family name of the contact
+ */
+
+/**
+ * ContactListTableMobile - Component for displaying contacts in a DataGrid
+ *
+ * @memberof Contacts
+ * @name ContactListTableMobile
+ * @param {object} Props - The props for ContactListTableMobile
+ * @param {Contact[]} Props.contacts - The list of contacts to display
+ * @param {Function} Props.deleteContact - Function to delete a contact
+ * @param {Function} Props.handleSendMessage - Function to handle sending a message
+ * @returns {React.JSX.Element} The ContactListTableMobile component
+ */
+const ContactListTableMobile = ({ contacts, deleteContact, handleSendMessage }) => {
+  const { addNotification } = useNotification();
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [openMenu, setOpenMenu] = useState(null);
+  const theme = useTheme();
+
+  const handleClick = (event, contact) => {
+    setAnchorEl(event.currentTarget);
+    setOpenMenu(contact.webId);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+    setOpenMenu(null);
+  };
+
+  const handleMenuItemClick = (action, contact) => () => {
+    action(contact);
+    handleClose();
+  };
+
+  const handleProfileClick = (contactData) => {
+    <ContactProfileIcon contact={contactData} />;
+  };
+
+  const iconSize = {
+    height: '24px',
+    width: '24px'
+  };
+
+  const iconStyling = {
+    width: '100%'
+  };
+
+  return (
+    <Box>
+      <Box
+        sx={{
+          my: '15px',
+          p: '15px',
+          background: theme.palette.primary.light,
+          color: '#fff',
+          borderRadius: '8px',
+          position: 'relative'
+        }}
+      >
+        <Box>
+          <Typography>Name</Typography>
+          <Box
+            sx={{
+              position: 'absolute',
+              top: '50%',
+              right: 15,
+              transform: 'translateY(-50%)'
+            }}
+          >
+            <Typography>Actions</Typography>
+          </Box>
+        </Box>
+      </Box>
+      {contacts?.map((contact) => (
+        <Box key={contact.webId}>
+          <Card
+            sx={{
+              my: '5px',
+              position: 'relative'
+            }}
+          >
+            <CardContent>
+              <Box>
+                <Typography variant="body1" component="div" noWrap>
+                  {contact.givenName || ''} {contact.familyName || ''}
+                </Typography>
+                <Typography variant="body2" component="div" noWrap color="text.secondary">
```

Comment:
I think adding a width to this Typography component (Like 90%) might be good in solving what Russell has pointed out.

![Screenshot 2024-06-25 at 9 53 19 PM](https://github.com/codeforpdx/PASS/assets/14917816/aa224c9c-dcdc-407b-a540-61152ee7f8a0)